src/js/components/DataFilters/stories/Inline.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Looks like we need to add labels to the search and sort by fields (see the accessibility tab in the story).

Reply:
done